---
layout: paper
title: "Monitoring single-cell gene regulation under dynamically controllable conditions with integrated microfluidics and software"
year: "2018"
shortref: "Kaiser M <i>et al. Nat Commun</i> 2018"
nickname:
journal: "Nature communications"
volume:
issue:
pages:
authors: "Matthias Kaiser, Florian Jug, Thomas Julou, Siddharth Deshpande, Thomas Pfohl, Olin K Silander, Gene Myers, Erik Van Nimwegen"
image: /assets/images/papers/default-paper.svg
redirect_from:
fulltext:
pdf:
pdflink:
github:
pmid: 29335514
pmcid:
f1000:
doi: "10.1038/s41467-017-02505-0"
dryad_doi:
figshare_doi:
altmetric_id:
category: paper
published: true
embargo: false
peerreview: true
review: false
tags: [microfluidics, gene expression]
---
{% include JB/setup %}

# Abstract

Much is still not understood about how gene regulatory interactions control cell fate decisions in single cells, in part due to the difficulty of directly observing gene regulatory processes in vivo. We introduce here a novel integrated setup consisting of a microfluidic chip and accompanying analysis software that enable long-term quantitative tracking of growth and gene expression in single cells. The dual-input Mother Machine (DIMM) chip enables controlled and continuous variation of external conditions, allowing direct observation of gene regulatory responses to changing conditions in single cells. The Mother Machine Analyzer (MoMA) software achieves unprecedented accuracy in segmenting and tracking cells, and streamlines high-throughput curation with a novel leveraged editing procedure. We demonstrate the power of the method by uncovering several novel features of an iconic gene regulatory program: the induction of Escherichia coliâ€™s lac operon in response to a switch from glucose to lactose.
