
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Blog</title>
    <meta name="description" content="The Drummond Lab blog">
    <meta name="author" content="Dr. Olin Silander">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>

    <!-- Bootstrap styles -->
    <link href="/assets/themes/lab/bootstrap/css/bootstrap.min.css" rel="stylesheet"/>
    <!-- Optional theme -->
    <link href="/assets/themes/lab/bootstrap/css/bootstrap-theme.min.css" rel="stylesheet"/>
    <!-- Sticky Footer -->
    <link href="/assets/themes/lab/bootstrap/css/bs-sticky-footer.css" rel="stylesheet"/>
    
    <!-- Custom styles -->
    <link href="/assets/themes/lab/css/style.css?body=1" rel="stylesheet" type="text/css" media="all"/>

    <!-- icon -->
    <link rel="shortcut icon" href="/assets/themes/lab/images/logo/favicon.ico"/>

    <!-- Custom fonts and icons via Font Awesome, http://fortawesome.github.io/Font-Awesome/ -->
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>

    <!-- Fonts via Google -->
    <link href='http://fonts.googleapis.com/css?family=Lato:300italic,700italic,300,700' rel='stylesheet' type='text/css'/>

    <!-- Math via MathJax -->
    <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>


    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
    <![endif]-->

    <!-- Fav and touch icons -->
    <!-- Update these with your own images
      <link rel="shortcut icon" href="images/favicon.ico">
      <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
      <link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
      <link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png">
    -->

    <!-- atom & rss feed
    <link href="/atom.xml" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="/rss.xml" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">
    -->

  </head>

  <body>

  <!-- Static top navbar -->

  <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class='container'>
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#jb-navbar-collapse">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a href="/">
        <img class="pull-left logo" src="/assets/themes/lab/images/logo/lab-logo.svg" type="image/svg+xml">
        </a>
      </div>

      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="jb-navbar-collapse">
        <ul class="nav navbar-nav navbar-left hidden-sm">
          <!-- Don't put site title on home page -- there's already a big title there. -->
          
          
          
          <a class="navbar-brand" href="/">Silander Lab</a>
          
        </ul>
        <ul class="nav navbar-nav navbar-right pull-right">
          
          
          


  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
  
    
  
    
      
    
  
    
  
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
      	
      	<li><a href="/projects/">Research</a></li>
      	
      
    
  
    
      
      	
      	<li><a href="/team/">Team</a></li>
      	
      
    
  
    
      
      	
      	<li><a href="/papers/">Papers</a></li>
      	
      
    
  
    
      
      	
      	<li class="active"><a href="/blog/" class="active">Blog</a></li>
      	
      
    
  
    
      
    
  
    
      
    
  



          <li class="dropdown nav navbar-nav">
            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
              More <span class="caret"></span>
            </a>

            <ul class="dropdown-menu nav navbar-nav" role="menu">
              
              
              


  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
  
    
  
    
      
    
  
    
  
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
    
  
    
      
      	
      	<li><a href="/protocols/">Protocols</a></li>
      	
      
    
  
    
      
      	
      	<li><a href="/news/">News</a></li>
      	
      
    
  



            </ul>
            </li>
        </ul>
      </div><!-- /.navbar-collapse -->
    </nav>
   </div>

   <!-- Content -->
   <div class="container">
      


<div class="pager">
  
	  
  
	  	
    
</div>



	

	<div class="bigspacer"></div>
	<div class="row">
		<div class="col-md-3">
			<div class="head">
				<a href="/blog/about-adaptive-phase-separation" class="off">Sticking together to survive stress</a>
			</div>
			<div class="smallnote">
				09 Mar 2017
								
			</div>	
		</div>
		<div class="col-md-8">		
			<div style="height:5px;"></div>			
			
			<div class="post">
				
<p>We’ve just published a <a href="/papers/paper/adaptive-phase-separation">paper</a> on how a protein’s ability to form hydrogels allows cells to sense and respond to stress.</p>

<h3 id="a-bit-of-background">A bit of background</h3>

<p>Like us, cells react to stresses like heat, starvation, and poison. All these stresses cause clumps of molecules to form inside cells. This clumping—aggregation—looks like damaged proteins sticking together. But in our last <a href="/papers/paper/endogenous-aggregates">paper</a>, we showed that instead, these clumps look like they’re a cellular response, not just signs of damage. (See our <a href="/blog/about-endogenous-aggregates">blog post</a> on it.)</p>

<p>We’re going to go back to the Cellular Institute with its manicured grounds and main building. Happy workers (proteins) go about their often inscrutable business.</p>

<p align="center">
<img src="/assets/images/cellular-institute-unstressed.png" width="50%" />
</p>

<p>The Institute is constantly dealing with awful changes in its environment. The mission is to survive, even to keep growing, during those tough times.</p>

<p align="center">
<img src="/assets/images/environmental-and-growth-reports.png" width="50%" />
</p>

<p>The picture that came out of our <a href="/blog/about-endogenous-aggregates">earlier study</a> suggested that, overall, the clumping in cells was an organized, reversible process, consistent with a coordinated effort to survive tough times.</p>

<p>But what about for specific proteins? How can you tell if clumping is stress-inflicted damage or an adaptive response? A key test is to see what happens when you stop the clumping during stress. Stopping damage during stress should make cells healthier. Stopping an adaptive stress response, on the other hand, would make cells sicker.</p>

<h3 id="doesnt-sound-too-complicated">Doesn’t sound too complicated</h3>

<p>Easy for you to say, headline! In our study, we had to first find a protein that formed clumps during stress, then figure out how to control its clumping without screwing up a bunch of other stuff (which involved quite a bit of figuring out why it formed clumps, physically, and which bits of the molecule were responsible).</p>

<p>Doing all that required working on the protein in isolation in a test tube, so then we had to put the protein back into cells  to see if it worked the same way. Finally, we could expose those cells to stress and see how they responded.</p>

<h3 id="what-did-we-do">What did we do?</h3>

<p>We picked out a specific protein that formed clumps during stress: poly(A)-binding protein, which is nicknamed Pab1 in baker’s yeast, our beautiful and hardy model organism. (Relatives of Pab1 are found in essentially every cell that has a nucleus, from plants to parasites to people.) We purified Pab1, put it in a dish, and exposed it to the same intracellular conditions that follow stress and are thought to cause clumping, heat and pH.</p>

<p>Yeast cells like to live in acidic (low-pH) environments, but keep their insides at neutral pH. When cells get stressed, their internal pH drops, becoming more acidic. That, others have argued, might cause clumping during non-heat stresses like starvation.</p>

<p align="center">
<img src="/assets/images/heat-starvation-ph-at-cell-institute.png" width="75%" />
</p>

<h3 id="what-did-we-find">What did we find?</h3>

<p>First, we discovered that Pab1 “clumping” was something rather specific and wonderful: <strong>phase separation</strong> into hydrogel droplets.</p>

<p>Phase separation is like a flash mob. Under certain conditions, suddenly, from a mixture of two types of molecules, one type surges together, separating from the other (“demixing”) like oil droplets from water. Bunches of recent studies have shown proteins doing this trick, usually under exotic conditions.</p>

<p align="center">
<img src="/assets/images/phase-separation-cartoon.png" width="75%" />
</p>

<p>Pab1 phase-separates in response to both pH and temperature, right at the stress-associated ranges of each one. It’s one of the coolest results in the paper: Pab1 is an extraordinarily sensitive <em>sensor of stress</em>, particularly temperature. You might be surprised to know that how yeast cells sense temperature has been a bit of mystery…</p>

<h3 id="back-into-the-cell">Back into the cell</h3>

<p>We figured out a way to make Pab1 phase separate at lower or higher temperatures. Remember our key test? That let us make cells with proteins that couldn’t get together during stress. And guess what we found?</p>

<p align="center">
<img src="/assets/images/disrupting-phase-separation.png" width="75%" />
</p>

<p>Cells with Pab1 molecules that didn’t stick together during stress couldn’t grow during stress. This happened during heat shock, and during starvation, too!</p>

<h3 id="ultimately-its-about-fitness">Ultimately, it’s about fitness</h3>

<p>There’s been a bunch of beautiful work on phase separation, hydrogel formation, and other quinary phenomena. (<strong>Quinary</strong>, a term introduced by Stuart Edelstein, refers to a step beyond traditional quaternary structures to structures involving undefined numbers of molecules.) But showing that these phenomena are actually good for cells, as opposed to just gorgeous curiosities, has been challenging. Pab1 delivered for us!</p>

<p align="center">
<img src="/assets/images/quinary-academy-certificate.png" width="75%" />
</p>

<h3 id="dive-in">Dive in!</h3>

<p>This <a href="/papers/paper/adaptive-phase-separation">paper</a> was five years from initial discovery to publication. There’s a lot in there (though the reviewers wanted oh so much more!). For experts, our results are full of surprises. First author <a href="/team/josh-riback">Josh</a> has blogged about some of the <a href="/blog/polymer-biophysics-in-action">biophysical aspects</a> and co-first author <a href="/team/chris-katanski">Chris</a> has blogged about the <a href="/blog/Pab1-Demixing-FAQ">in vivo aspects</a>.</p>

<p>And there’s much more to do. How, exactly, does Pab1’s phase separation help the cell? We share some ideas in the paper. How, exactly, does Pab1 sense temperature and pH? Watch this space as the story continues to, well, come together.</p>


			</div>
		</div>			
		<div class="col-md-1"></div>
	</div>		



	

	<div class="bigspacer"></div>
	<div class="row">
		<div class="col-md-3">
			<div class="head">
				<a href="/blog/Pab1-Demixing-FAQ" class="off">Behind the scenes of Pab1 demixing</a>
			</div>
			<div class="smallnote">
				09 Mar 2017
								
			</div>	
		</div>
		<div class="col-md-8">		
			<div style="height:5px;"></div>			
			
			<div class="post">
				<p>Hi, I’m Chris. I’m a student in the Drummond lab and coauthor on the <a href="/papers/paper/adaptive-phase-separation">paper</a> “<em>Stress-Triggered Phase Separation Is an Adaptive, Evolutionarily Tuned Response</em>.” When presenting this work, I’ve been asked a few questions repeatedly that I will answer in a <strong>FAQ</strong> section. I’ll also share an anecdote about the challenges of delineating Pab1’s role in the stress response from its many other functions. I hope the story emphasizes the importance of <strong>endogenous protein expression levels</strong>.</p>

<h1 id="faqs">FAQs:</h1>

<h3 id="the-δp-yeast-strain-grows-better-at-high-temperature-than-the-other-demixing-impaired-mutants--what-gives"><em>The ΔP yeast strain grows better at high temperature than the other demixing-impaired mutants – what gives?</em></h3>
<p>How can a few point mutations in the P-domain have a greater fitness effect than deleting the domain wholesale? In <strong>figure 6F</strong>, when we compare the growth at 40° C of yeast expressing Pab1 variants; mutants “MV-&gt;A” and “MVFY-&gt;AGPNQ” show a major growth defect. These mutants show a greater defect than the strain expressing Pab1ΔP. This is result is most easily rationalized through the lens of demixing: Pab1ΔP demixes more robustly than either of the mutants; therefore it grows more robustly at high temperature. <strong>Figure 6G</strong> illustrates the relation between growth and Pab1 demixing. The related question, “how can a few point mutations have a greater effect on demixing than deleting the domain wholesale?” remains open. Josh discusses this point in greater detail in <a href="/blog/polymer-biophysics-in-action">his blog post</a>.</p>

<h3 id="how-do-mutations-to-the-p-domain-affect-stress-granule-formation"><em>How do mutations to the P-domain affect stress granule formation?</em></h3>
<p>I don’t know. The stress used in <strong>figure 6D</strong> to trigger Pab1 demixing is too mild to elicit Pab1-marked stress granules in wild type cells (<strong>figure 1A</strong>). Moreover, while stress granules are easily observed, it is not clear that they are the physiologically functional species. Here, we make the case that perturbing demixing of Pab1 is sufficient to affect physiology. In fact, I favor the hypothesis that quinary assemblies and stress granules are related, but separable phenomena. Pab1 forms sedimentable quinary assemblies in the absence of visible focus formation. Conversely, HSP104 is an example of a protein that colocalizes with stress granules, but isn’t sedimentable.</p>
<p align="center">
<img src="/assets/images/CK_demix_then_SG_blueprint.png" width="75%" />
</p>

<h3 id="is-the-assembly-of-other-proteins-perturbed"><em>Is the assembly of other proteins perturbed?</em></h3>
<p>Yef3 is another protein that forms sedimentable species during mild stress conditions. <strong>Figure S5</strong> indicates that demixing of Yef3 does not vary between P-domain mutants. However, it is possible that demixing of one of the other ~150 heat-assembling proteins is affected. At its core, this question is about the mechanism of formation of large cellular assemblies. Are all proteins demixing autonomously, or can one protein recruit another to these structures? Is RNA binding sufficient to recruit some proteins? Is recruitment stress specific?</p>

<h3 id="what-is-going-on-with-rna"><em>What is going on with RNA?</em></h3>
<p>Right? In vitro we show the following: RNA is not necessary for Pab1 demixing (<strong>fig 2E</strong>), RNA actually inhibits demixing (<strong>fig 2C</strong>); and short poly(A) sequences are released during demixing (<strong>fig 2A</strong>). One interpretation is that demixing is dominated by protein-protein interactions and these interactions are mutually exclusive with RNA binding. In vivo we show that heat-triggered sedimentation of Pab1 is RNase1 insensitive (<strong>fig 1B</strong>). This result is consistent with the in vitro results, though other interpretations are possible. This model predicts that demixing of Pab1 and poly(A)-positive RNA are not mechanistically connected. I find this unintuitive, but testable.</p>

<h3 id="you-speculate-about-pab1-as-a-translational-regulator-of-the-stress-response-how-much-is-speculation"><em>You speculate about Pab1 as a translational regulator of the stress response. How much is speculation?</em></h3>
<p>If you’re curious about the mechanism by which Pab1 demixing affects stress tolerance, then come to my talk at <strong>ASBMB</strong> “<em>Translation of heat shock proteins is regulated by poly(A)-binding protein assembly</em>.” Its on <strong>Sunday April 23 2017</strong> during the <strong>Protein folding, Aggregation, and Chaperones session</strong>; or come see the poster later.</p>

<h1 id="make-happy-yeast">Make happy yeast:</h1>
<p>Pab1 is an essential gene with a role in many cellular processes. If one wishes to make mutations, delete then replace isn’t an option. Other work has successfully expressed Pab1 from a plasmid. However, cells are very sensitive to Pab1 expression levels – consistent with its known autoregulatory expression mechanism. My first attempt to modify the Pab1 P-domain was with a construct that replaced the native genomic 3’ UTR with a KanMX selectable marker. I also made the control strain with the same marker, but preserved P-domain. The control strain grew slowly and at 30°C and expressed Pab1 at ~20% the wild type. Reduced expression is problematic because demxing is concentration dependent; similarly, over-expression can lead to demixing artifacts. Worse, the proteome of this control strain resembled that of stressed wild type cells, including high chaperone expression. Ultimately, I developed the serial transformation strategy described in the methods (and another blog post). The resulting strains have no selectable markers, native 3’ UTRs, and no extra nucleotide scars. This manor of strain construction solved the reduced Pab1 expression and slow growth problems. Now, I am able to discern the relation of assembly and the stress response, without conflation of growth rate effects.</p>
<p align="center">
<img src="/assets/images/CK_cool_yeast.png" width="75%" />
</p>


			</div>
		</div>			
		<div class="col-md-1"></div>
	</div>		



<div class="bigspacer"></div>	
<div class="row">
	<div class="col-md-12 text-center">
		<a class="off" href="//archive/">
			<div class="smallhead">Archive</div>
			<i class="text-gray fa fa-chevron-circle-down fa-2x"></i>
		</a>
	</div>
</div>
   </div>


  <!-- Static bottom navbar -->
  <nav class="navbar navbar-default navbar-fixed-bottom">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#jb-bottom-collapse"> <!-- NOTE! data-target was changed to jb-bottom-collapse -->
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
      </div>
      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="jb-bottom-collapse">
        <ul class="nav navbar-nav navbar-left navbar-small">
          <li><a href=http://www.massey.ac.nz/massey/explore/departments/institute-natural-mathematical-sciences/institute-natural-mathematical-sciences_home.cfm>Institute of Natural and Mathematical Sciences</a></li>
          <li><a href=http://www.massey.ac.nz/>Massey University</a></li>
        </ul>
        <ul class="nav navbar-nav navbar-right navbar-small">
          <li><a href='/about.html#purpose'>About</a></li>
          <li><a href='/about.html#join'>Join</a></li>
          <li><a href='/about.html#contact'>Contact</a></li>
        </ul>
      </div><!-- /.navbar-collapse -->
    </div>
  </nav>
 


    




  <script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-62008526-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>





    <!-- Latest compiled and minified JavaScript, requires jQuery 1.x (2.x not supported in IE8) -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
    <script src="/assets/themes/lab/bootstrap/js/bootstrap.min.js"></script>
  </body>
</html>

